.github/workflows/build-docker-corsika-simtelarray-image.yml

```diff
@@ -205,3 +210,52 @@ jobs:
           file: ./docker/Dockerfile-prod-opt
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}-prod-opt
+      - name: set image tag
+        run: |
+            echo "image_tag=$(echo '${{ steps.meta.outputs.tags }}' | head -n1)" >> "$GITHUB_ENV"
+        id: set-image-tag
+    outputs:
+      image_tag: ${{ steps.meta.outputs.tags }}
+
+  test-images:
+    runs-on: ubuntu-latest
+    needs: build-production-images-optimized
+    strategy:
+      matrix:
+        version:
+          - {simtel_version: '240205', corsika_version: '77500', bernlohr_version: '1.67'}
+          # - {simtel_version: '240927', corsika_version: '77550', bernlohr_version: '1.68'}
+        hadronic_model: ['qgs2']
+        production:
+          - {build_opt: 'prod6-sc', extra_def: '-DMAXIMUM_TELESCOPES=128'}
+          - {build_opt: 'prod6-baseline', extra_def: ''}
+        avx_instruction: ['no_opt']
+    container:
+      image: ghcr.io/gammasim/simtools-prod-opt-${{ matrix.version.simtel_version }}-corsika-${{ matrix.version.corsika_version }}-bernlohr-${{ matrix.version.bernlohr_version }}-${{ matrix.production.build_opt }}-${{ matrix.hadronic_model }}-${{ matrix.avx_instruction }}
+      env:
+        SIMTOOLS_SIMTEL_PATH: "/workdir/sim_telarray"
+        SIMTOOLS_DB_API_PORT: ${{ env.SIMTOOLS_DB_API_PORT }}
+        SIMTOOLS_DB_API_USER: ${{ env.SIMTOOLS_DB_API_USER }}
+        SIMTOOLS_DB_API_PW: ${{ env.SIMTOOLS_DB_API_PW }}
+        SIMTOOLS_DB_SERVER: ${{ env.SIMTOOLS_DB_SERVER }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set sim_telarray path
+        run: |
+          echo "PATH=\$PATH:/usr/bin:/usr/local/bin:$SIMTOOLS_SIMTEL_PATH" >> "$GITHUB_ENV"
```

Comment:
Do you really need to explicitly set `/usr/bin:/usr/local/bin`? There's no harm in it of course, but if it's not needed I would remove it (who knows, maybe in a future OS it will change).

Reply:
Yes, needed - running the workflow without it will give you:

```
OCI runtime exec failed: exec failed: unable to start container process: exec: "sh": executable file not found in $PATH: unknown
```